assets/js/blocks/checkout/inner-blocks/checkout-billing-address-block/frontend.tsx

```diff
@@ -43,7 +49,19 @@ const FrontendBlock = ( {
 	if ( ! showBillingFields && ! forcedBillingAddress ) {
 		return null;
 	}
-
+	if ( forcedBillingAddress ) {
```

Comment:
Please see my comment about `if/else` and ternary operators in the `edit.tsx` file and consider if the same applies here.